warehouse/bigquery/bigquery.go

```diff
@@ -300,12 +301,12 @@ func (bq *HandleT) loadTable(tableName string, _, getLoadFileLocFromTableUploads
 
 	loadTableByAppend := func() (err error) {
 		stagingLoadTable.partitionDate = time.Now().Format("2006-01-02")
-		outputTable := tableName
+		outputTable := partitionedTable(tableName, stagingLoadTable.partitionDate)
 		// Tables created by RudderStack are ingestion-time partitioned table with pseudo column named _PARTITIONTIME. BigQuery automatically assigns rows to partitions based
 		// on the time when BigQuery ingests the data. To support custom field partitions, omitting loading into partitioned table like tableName$20191221
 		// TODO: Support custom field partition on users & identifies tables
-		if !customPartitionsEnabled {
-			outputTable = partitionedTable(tableName, stagingLoadTable.partitionDate)
+		if customPartitionsEnabled || misc.Contains(customPartitionsEnabledWorkspaceIDs, bq.warehouse.WorkspaceID) {
```

Comment:
You can use [slice.Contains](https://pkg.go.dev/golang.org/x/exp/slices#Contains
)
```suggestion
		if customPartitionsEnabled || slices.Contains(customPartitionsEnabledWorkspaceIDs, bq.warehouse.WorkspaceID) {
```